tools/common/Frameworks.cs

```diff
@@ -280,7 +281,6 @@ public static Frameworks GetiOSFrameworks (Application app)
 
 				{ "CallKit", "CallKit", 10 },
 				{ "Messages", "Messages", 10 },
-				{ "Speech", "Speech", 10 },
```

Comment:
same, drop the iOS related changes in this file, once it is added it does not need to be changed again, just keep the Mac addition

Reply:
why is this resolved ? that framework was already in iOS and should not be changed to `13,0`